package-lock.json

```diff
@@ -690,108 +689,6 @@
         "url": "https://github.com/sponsors/isaacs"
       }
     },
-    "node_modules/@nx/nx-darwin-arm64": {
```

Comment:
i'm concerned about deleting package-lock and then running a npm install for this case.
When we add/remove dependencies or update a major version of a dependency, we should not delete package-lock. Deleting will cause installs of latest minor versions of other dependencies and we did not trace any changelog/breaking changes for those. 
Please reinstate original package-lock, perform an npm i and then push the file.